sdks/java/io/kafka/src/main/java/org/apache/beam/sdk/io/kafka/TopicPartitionCoder.java

```diff
@@ -48,7 +49,7 @@ public TopicPartition decode(InputStream inStream) throws CoderException, IOExce
 
   @Override
   public List<? extends Coder<?>> getCoderArguments() {
-    return null;
+    return ImmutableList.of(StringUtf8Coder.of(), VarIntCoder.of());
```

Comment:
Why it's needed?

Reply:
It is used when using the coder in state, which is invoked by `StructureCoder.toString()`. That's why I added a unit test for that.